articles/integrations/azure-api-management/configure-auth0.md

```diff
@@ -81,4 +81,4 @@ At this point, you've set up Auth0 for use as an OAuth 2.0 authorization server.
 
 <%= include('./_stepnav', {
  prev: ["Integrate Azure API Management Service with Auth0", "/integrations/azure-api-management"], next: ["2. Configure Azure API Management", "/integrations/azure-api-management/configure-azure"]
-}) %>
```

Comment:
I don't know why this line appears changed, seems to be the same 🤔 , I edited the file in Github editor.